vivisect/__init__.py

```diff
@@ -1112,8 +1113,7 @@ def isProbablyCode(self, va, **kwargs):
     #
     # Opcode API
     #
-    @functools.cache
-    def parseOpcode(self, va, arch=envi.ARCH_DEFAULT):
+    def parseOpcode(self, va, arch=envi.ARCH_DEFAULT, skip=False):
```

Comment:
```suggestion
    def parseOpcode(self, va, arch=envi.ARCH_DEFAULT, skip_cache=False):
```

Reply:
i'm with @williballenthin , the naming bothered me too.

Reply:
I prefer shorter names, but fair.